rpcserverhelp.go

```diff
@@ -488,6 +488,17 @@ var helpDescsEnUS = map[string]string{
 	"setgenerate-generate":     "Use true to enable generation, false to disable it",
 	"setgenerate-genproclimit": "The number of processors (cores) to limit generation to or -1 for default",
 
+	// AddMiningAddrCmd help.
+	"addminingaddr--synopsis": "Adds an additional available mining address to the set of mining addresses for the daemon",
+	"addminingaddr-address":   "The address to append to the set of mining addresses",
+
+	// ListMiningAddrsCmd help.
+	"listminingaddrs--synopsis": "Returns a list of the current active mining addresses",
+
```

Comment:
```Go
	"listminingaddrs--result0":  "List of current active mining addresses",
```

Otherwise:

```
$ btcctl help listminingaddrs
...
Result:
Nothing
```